From 7155246f1d3aa2bea20bd0b5404578f773e52c18 Mon Sep 17 00:00:00 2001
From: Marc Titinger <mtitinger@baylibre.com>
Date: Thu, 27 Aug 2015 15:40:49 +0200
Subject: [PATCH 09/11] omap: genpd: hack PM_GENERIC_DOMAIN into omap4

Signed-off-by: Marc Titinger <mtitinger@baylibre.com>
---
 arch/arm/plat-omap/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/plat-omap/Kconfig b/arch/arm/plat-omap/Kconfig
index d055db3..323838e 100644
--- a/arch/arm/plat-omap/Kconfig
+++ b/arch/arm/plat-omap/Kconfig
@@ -24,6 +24,7 @@ config POWER_AVS_OMAP
 	bool "AVS(Adaptive Voltage Scaling) support for OMAP IP versions 1&2"
 	depends on POWER_AVS && (ARCH_OMAP3 || ARCH_OMAP4) && PM
 	select POWER_SUPPLY
+	select PM_GENERIC_DOMAINS if PM
 	help
 	  Say Y to enable AVS(Adaptive Voltage Scaling)
 	  support on OMAP containing the version 1 or
-- 
1.9.1

