--- include/configs/vexpress_aemv8a.h_SAVE	2016-01-13 17:34:30.034145999 +0100
+++ include/configs/vexpress_aemv8a.h	2016-01-13 17:17:30.251073079 +0100
@@ -8,9 +8,6 @@
 #ifndef __VEXPRESS_AEMV8A_H
 #define __VEXPRESS_AEMV8A_H
 
-
-#define CONFIG_TARGET_VEXPRESS64_JUNO 1
-
 /* We use generic board and device manager for v8 Versatile Express */
 #define CONFIG_SYS_GENERIC_BOARD
 #define CONFIG_DM
@@ -204,37 +201,20 @@
 				"fdt_addr=0x83000000\0" \
 				"fdt_high=0xffffffffffffffff\0" \
 				"initrd_high=0xffffffffffffffff\0" \
-				"netmask=255.255.255.0\0"\
-				"serverip=SERVERIP\0"\
-				"bootserver=SERVERIP\0"\
-				"ipaddr=BOARDIP\0"
 
 /* Assume we boot with root on the first partition of a USB stick */
-//#define CONFIG_BOOTARGS		"console=ttyAMA0,115200n8 " \
+#define CONFIG_BOOTARGS		"console=ttyAMA0,115200n8 " \
 				"root=/dev/sda1 rw " \
 				"rootwait "\
 				"earlyprintk=pl011,0x7ff80000 debug user_debug=31 "\
 				"loglevel=9"
 
-#define CONFIG_BOOTARGS		"console=ttyAMA0,115200n8 " \
-				"earlyprintk=pl011,0x7ff80000 debug " \
-				"loglevel=9 " \
-				"root=/dev/nfs " \
-				"ip=BOARDIP " \
-				"nfsroot=SERVERIP:INSTALL_MOD_PATH rw " \
-				"no_console_suspend " \
-				"nfsrootdebug "
-
 /* Copy the kernel and FDT to DRAM memory and boot */
-//#define CONFIG_BOOTCOMMAND	"afs load ${kernel_name} ${kernel_addr} ; " \
+#define CONFIG_BOOTCOMMAND	"afs load ${kernel_name} ${kernel_addr} ; " \
 				"afs load  ${fdt_name} ${fdt_addr} ; " \
 				"fdt addr ${fdt_addr}; fdt resize; " \
 				"booti ${kernel_addr} - ${fdt_addr}"
 
-#define CONFIG_BOOTCOMMAND	"tftpboot 0x80000000 Image ; "\
-				"tftpboot 0x83000000 juno.dtb; "\
-				"booti 0x80000000 - 0x83000000"
-
 #define CONFIG_BOOTDELAY		1
 
 #elif CONFIG_TARGET_VEXPRESS64_BASE_FVP
