From f2400a9cd49ba418738c760e28f97cead2548d83 Mon Sep 17 00:00:00 2001
From: Axel Haslam <ahaslam@baylibre.com>
Date: Mon, 23 Mar 2015 11:06:29 +0100
Subject: [PATCH] bootargs axel

Signed-off-by: Axel Haslam <ahaslam@baylibre.com>
---
 include/configs/vexpress_aemv8a.h | 36 +++++++++++++++++++++++++++---------
 1 file changed, 27 insertions(+), 9 deletions(-)

diff --git a/include/configs/vexpress_aemv8a.h b/include/configs/vexpress_aemv8a.h
index 2cbbdef..b37e7b3 100644
--- a/include/configs/vexpress_aemv8a.h
+++ b/include/configs/vexpress_aemv8a.h
@@ -215,22 +215,37 @@
 				"max_fdt=0x100000\0" \
 				"fdt_high=0xffffffffffffffff\0" \
 				"initrd_high=0xffffffffffffffff\0" \
+				"serverip=192.168.10.3\0"\
+				"bootserver=192.168.10.3\0"\
+				"ipaddr=192.168.10.4\0"
 
 /* Assume we boot with root on the first partition of a USB stick */
 #define CONFIG_BOOTARGS		"console=ttyAMA0,115200n8 " \
-				"root=/dev/sda1 rw " \
-				"earlyprintk=pl011,0x7ff80000 debug user_debug=31 "\
-				"loglevel=9"
+				"root=/dev/nfs " \
+				"ip=192.168.10.4:192.168.10.3:192.168.10.3:255.255.255.0::::: " \
+				"nfsroot=192.168.10.3:/home/axelh/rootfs64 rw " \
+				"rdinit=/init "\
+				"root-dir=/home/axelh/rootfs64 "\
+				"server-ip=192.168.10.3 "\
+				"nfsrootdebug "\
+				"loglevel=7 "\
+				"no_console_suspend "
 
 /* Copy the kernel and FDT to DRAM memory and boot */
+#if 1
+#define CONFIG_BOOTCOMMAND	"tftpboot 0x80000000 Image ; "\
+				"tftpboot 0x83000000 juno.dtb; "\
+				"booti 0x80000000 - 0x83000000"
+#else
 #define CONFIG_BOOTCOMMAND	"cp ${kernel_addr} ${kernel_addr_r} ${max_kernel} ; " \
 				"cp ${fdt_addr} ${fdt_addr_r} ${max_fdt}; " \
 				"fdt addr $fdt_addr_r; fdt resize; " \
 				"booti $kernel_addr_r - $fdt_addr_r"
-
-#define CONFIG_BOOTDELAY		1
+#endif
+#define CONFIG_BOOTDELAY		3
 
 #elif CONFIG_TARGET_VEXPRESS64_BASE_FVP
+#error 2
 #define CONFIG_EXTRA_ENV_SETTINGS	\
 				"kernel_name=uImage\0"	\
 				"kernel_addr_r=0x80000000\0"	\
@@ -252,16 +267,19 @@
 #define CONFIG_BOOTDELAY		1
 
 #else
-
+#error 1
 #define CONFIG_EXTRA_ENV_SETTINGS	\
 					"kernel_addr_r=0x80000000\0"	\
 					"initrd_addr_r=0x88000000\0"	\
 					"fdt_addr_r=0x83000000\0"		\
-					"fdt_high=0xa0000000\0"
+					"fdt_high=0xa0000000\0"\
+					"netmask=255.255.255.0\0"\
+					"serverip=192.168.10.3\0"
+					"ipaddr=192.168.10.4\0"
+
 
 #define CONFIG_BOOTARGS			"console=ttyAMA0,115200n8 root=/dev/ram0"
-#define CONFIG_BOOTCOMMAND		"bootm $kernel_addr_r " \
-					"$initrd_addr_r:$initrd_size $fdt_addr_r"
+#define CONFIG_BOOTCOMMAND		"tftpboot 0x80000000 Image; tftpboot 0x83000000 juno.dtb; booti 0x80000000 - 0x83000000" \
 #define CONFIG_BOOTDELAY		-1
 #endif
 
-- 
2.4.5

